src/mesh/Mesh.hpp

```diff
@@ -69,12 +69,27 @@ class Mesh {
       int         dimensions,
       MeshID      id);
 
+  /// Mutable access to a vertex by VertexID
+  Vertex &vertex(VertexID id);
+
+  /// Const access to a vertex by VertexID
+  const Vertex &vertex(VertexID id) const;
```

Comment:
Does it make sense to delete the `operator[]`, to enforce that people use the checked variant?

Reply:
The operator[] is part of the std::deque so deleting is only an option if we want to hide vertices() somehow. Possible, but that's a bigger refactoring.

Reply:
Opened an issue https://github.com/precice/precice/issues/1971